README.md

```diff
@@ -3,69 +3,73 @@
 [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
 # Knife CLC
-
-This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like datacenters, templates and groups.
+This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like data centers, templates and groups.
 
 ## Installation
+There are three ways to install the Chef Knife plugin on your CenturyLink Cloud platform.
 
-If you're using [ChefDK](https://downloads.chef.io/chef-dk/), simply install the Gem:
+* If you're using [ChefDK](https://downloads.chef.io/chef-dk/), simply install the Gem:
 
 ```bash
 $ chef gem install knife-clc
 ```
 
-If you're using Bundler, add this line to your application's Gemfile:
+* If you're using Bundler:
+
+Add this line to your application's Gemfile:
 
 ```ruby
 gem 'knife-clc'
 ```
 
-And then execute:
+And then, execute:
 
 ```bash
 $ bundle
 ```
 
-Or install it yourself as:
+* If you're installing it yourself:
 
 ```
 $ gem install knife-clc
 ```
 
 ## Configuration
-In order to use CLC API, user must supply API username & password. This can be done in several ways:
+In order to use the CLC API, which runs the Chef Knife plugin, a user must supply an API username & password. This can be done in several ways.
```

Comment:
Wording is a bit incorrect here. Knife plugin uses CLC API under the hood, not the other way around. So we need to rephrase it somehow.